docs/bots-guide.md

```diff
@@ -109,12 +109,15 @@ You need:
     * Now, the bot can do its job on the streams you subscribed it to.
     * (In future versions of the API, this step may not be required).
 
-5. Run the bot.
+5. Run the bot. (There are two ways to run your bot.)
```

Comment:
Instead of saying this here, I'd add a very quick explanation after each command as to what it does.